### PR TITLE
Issue checkstyle#13213: remove //ok from input files

### PR DESCRIPTION
**Pull request by akankshat05**
Issue #13213 
Removed //ok from 14 input files.

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule232specialescape/InputIllegalTokenText.java [hunk 1]
```diff
@@ -14,32 +14,32 @@ public void methodWithLiterals()
...
     {
-        String r1 = "\b"; //ok
-        String r2 = "\t"; //ok
-        String r3 = "\n"; //ok
-        String r4 = "\f"; //ok
-        String r5 = "\r"; //ok
-        String r6 = "\""; //ok
-        String r7 = "\'"; //ok
-        String r8 = "\\"; //ok
+        String r1 = "\b";
```
Comment (sktpy) [inline src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule232specialescape/InputIllegalTokenText.java:30]:
```suggestion
        String r1 = "\b"; 
```
There shouldn't be any trailing white-space at the end of line after removing the comment. Apply the changes for all such occurrence.

**Comment by sktpy** [comment]
- Please keep one PR related to a single module only.
- Suppression for the corresponding comments should be also removed from the [file](https://github.com/checkstyle/checkstyle/blob/b6301c189ef5d2d731a755fa9a2e84aa0677494a/config/checkstyle-input-suppressions.xml#L68-L70), after the removal of comments from the .
- Please go through the issue description and the merged PRs for a better understanding regarding the changes.
- Make sure `mvn clean verify` passes locally before pushing the changes.

File: src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule21filename/InputOuterTypeFilename1.java [hunk 2]
```diff
@@ -1,17 +1,17 @@
 package com.google.checkstyle.test.chapter2filebasic.rule21filename;
```
Comment (MANISH-K-07) [inline src/it/resources/com/google/checkstyle/test/chapter2filebasic/rule21filename/InputOuterTypeFilename1.java:1]: It is required to remove the related suppression from the file  [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml). 
Search for the filename in the mentioned file and delete that respective suppression. They are all organized directory wise, so it should be easy to browse.
Repeat the same for all 14 files that have been modified.

**Review by MANISH-K-07 — COMMENTED** [review]
Hey @akankshat05 ,
After you have made the changes suggested by @sktpy , 
make sure to remove the suppressions  also as mentioned so that the authenticity of your work is verified and next contributors don't get confused.

**Comment by akankshat05** [comment]
@MANISH-K-07 @sktpy Im currently making the necessary changes, but I could not find the suppressions for a few files like - InputOneTopLevelClassBad1

**Comment by MANISH-K-07** [comment]
> @MANISH-K-07 @sktpy Im currently making the necessary changes, but I could not find the suppressions for a few files like - InputOneTopLevelClassBad1

The files you chose to modify are not from the path `src/test`
Only input files related to this path have suppressions in the mentioned file.
@sktpy , do you have any idea if and where the suppressions related `src/it ` are located ?....

Try picking any module from said file below this comment : 
`<!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->`

**Comment by akankshat05** [comment]
> > @MANISH-K-07 @sktpy Im currently making the necessary changes, but I could not find the suppressions for a few files like - InputOneTopLevelClassBad1
> 
> The files you chose to modify are not from the path `src/test` Only input files related to this path have suppressions in the mentioned file. @sktpy , do you have any idea if and where the suppressions related `src/it ` are located ?....
> 
> @akankshat05 , Or you could try picking any module from said file since you are new to the project Search below this comment : `<!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->`

Noted, I am only modifying checks/whitespace/whitespacearound files.

**Comment by MANISH-K-07** [comment]
> Noted, I am only modifying checks/whitespace/whitespacearound files.

try to limit the number of files that you are modifying.
This makes it easier to debug.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundKeywordsAndOperators.java [hunk 3]
```diff
@@ -156,10 +156,10 @@ private void divTest()
     /** assert statement test */
     public void assertTest()
     {
-        // OK
+
         assert true;
 
-        // OK
+
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundKeywordsAndOperators.java:162]: One blank line should be enough.
Please remove the extra newline

**Review by MANISH-K-07 — COMMENTED** [review]
Please make sure that your commit message adheres to Checkstyle rules : 

```
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit b658d10e43038a937dbb69b3e76266959c2d4022 message: "Issue checkstyle#13213: Removed //ok from whitespacearound\n" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was: 1

```

File: config/checkstyle-input-suppressions.xml [hunk 4]
```diff
@@ -2742,16 +2742,6 @@
             files="checks[\\/]whitespace[\\/]whitespaceafter[\\/]InputWhitespaceAfterWithEmoji.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]whitespace[\\/]whitespaceafter[\\/]InputWhitespaceAfterWithEmoji.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]whitespacearound[\\/]InputWhitespaceAround1.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]whitespacearound[\\/]InputWhitespaceAround3.java"/>
-  <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:2749]: lgtm....

**Review by MANISH-K-07 — COMMENTED** [review]
Great !!
All CI are green. Just give it some time for approval
👍🏻

**Comment by sktpy** [comment]
@akankshat05 Why have placed the `ok` comments for the file `InputWhitespaceAroundKeywordsAndOperators` again in the [new commit](https://github.com/checkstyle/checkstyle/compare/b658d10e43038a937dbb69b3e76266959c2d4022..81cf410377f4e5fc413337a0ad300fd3af3949a3)? They are also the part of `whitespacearound`.
Please remove them again, and update the PR description and commit message to specify the module name i.e. `whitespacearound`. 

Also, please add the link to documentation for the corresponding module in the PR description.
The documentation isn't really necessary for this specific PR but having the habbit to do such is not only beneficial in making you not forget to place them in subsequent PRs which require them, but will get you practiced in navigating the documentation.
Apart from that, when you go look for the documentation related to this module, having a glance at it will increase your knowledge regarding Checkstyle steadily.

**Comment by akankshat05** [comment]
If I remove the comments from the InputWhitespaceAroundKeywordsAndOperators file, I am getting the below checkstyle errors after mvn clean verify :
missing (8)
#1      : 169:9: 'assert' is not followed by whitespace.
#2      : 172:20: ':' is not preceded with whitespace.
#3      : 172:20: ':' is not followed by whitespace.
#4      : 278:13: '}' is not followed by whitespace. 
#5      : 307:24: '+' is not followed by whitespace.
#6      : 307:24: '+' is not preceded with whitespace.
#7      : 307:28: '+' is not followed by whitespace.
#8      : 307:28: '+' is not preceded with whitespace.

unexpected (8)
#1      : 168:9: 'assert' is not followed by whitespace.
#2      : 171:20: ':' is not followed by whitespace.
#3      : 171:20: ':' is not preceded with whitespace.
#4      : 277:13: '}' is not followed by whitespace.
#5      : 306:24: '+' is not followed by whitespace.
#6      : 306:24: '+' is not preceded with whitespace.
#7      : 306:28: '+' is not followed by whitespace.
#8      : 306:28: '+' is not preceded with whitespace.

**Comment by MANISH-K-07** [comment]
@akankshat05 
The errors you are getting are because of mismatched violation lines. Adjusting the expected violation list from the respective check ought to solve this. However, I would not recommend working on this as part of this PR.
The joy and satisfaction that a first successful merge in any organization gives is immense and I respect that.

Working on the above mentioned changes might just be overwhelming for a first PR, causing troubles if the code isn't properly understood and I wouldn't want you to regret contributing to this wonderful organization. **`;)`**

Instead, I would recommend you to work on a few more good first issues and once you feel you are ready, you can always get back to this input file again and finish what you have started....
@sktpy , do you agree ?..

**Comment by sktpy** [comment]
@MANISH-K-07 @akankshat05 The errors are due to the removal of the whole line from here https://github.com/checkstyle/checkstyle/pull/14422#discussion_r1478419235 because it only contained the `ok` comment,
which decreased the actual line numbers for those violations by 1.

Apart from that there is an additional comment like that whose removal will also shift the actual line number for violations by 2 in total.

Granted that changing the Check test in this PR is out of scope, but it wouldn't be too difficult to decrease the line numbers for those 8 violations of expected array and decrease them by 2.
We can guide on how will you do that, which will be beneficial as the other good first issue (`Enforce file size on Java input`) involves these changes as well.

**Comment by MANISH-K-07** [comment]
> We can guide you on this, which will be beneficial as the other good first issue (Enforce file size on Java input) involves these changes as well.

@akankshat05 , Do you think you can proceed with this ?
It is completely fine if you want to take time instead.

**Comment by akankshat05** [comment]
@MANISH-K-07 @sktpy Sure, I will proceed with fixing the errors , then continue back on this PR.

**Comment by MANISH-K-07** [comment]
> @MANISH-K-07 @sktpy Sure, I will proceed with fixing the errors , then continue back on this PR.

Well then ....

You will need to identify the violation lines from modified input file and change accordingly in the expected string of the respective test method in  the file [WhitespaceAroundCheckTest](https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheckTest.java).

Like @sktpy mentioned, I think all the violations jump up two lines. 
Don't take our word for it. Check where the violations are and then modify.
make sure to run using `mvn clean verify` after your commit before pushing up the changes

Ping in case you have any queries
Good luck  ;)

**Comment by sktpy** [comment]
> Like @sktpy mentioned, I think all the violations jump up two lines.

This will require removal of the extra line which wasn't done earlier.
<br>

@akankshat05 For a better understanding, you can also go through the merged PRs of the issue that I have referenced above, and focus on how splitting the CheckInputs changed the line numbers of the violations, which required changes in the corresponding CheckTest.

**Comment by MANISH-K-07** [comment]
@akankshat05 
Please mark the PR as draft until you are done working on it....

**Comment by nrmancuso** [comment]
@akankshat05 how can we help to get this PR out of draft mode?

**Comment by romani** [comment]
if no reply in 2 week, PR will be marked as abandoned.

**Comment by nrmancuso** [comment]
We have lost contact with author, anyone is welcome to reuse changes here to close the issue.

**Pull request closed without merging**